src/Sentry/SentryHttpFailedRequestHandler.cs

```diff
@@ -0,0 +1,79 @@
+using Sentry.Internal;
+using Sentry.Protocol;
+
+namespace Sentry;
+
+internal class SentryHttpFailedRequestHandler : SentryFailedRequestHandler
+{
+    public const string MechanismType = "SentryHttpFailedRequestHandler";
+
+    internal SentryHttpFailedRequestHandler(IHub hub, SentryOptions options)
```

Comment:
```suggestion
    public SentryHttpFailedRequestHandler(IHub hub, SentryOptions options)
```

Reply:
That one [was/is internal in our main branch as well](https://github.com/getsentry/sentry-dotnet/blob/d4dda091cd4bc957d2b5f582fd3b3e83f9b157ab/src/Sentry/SentryFailedRequestHandler.cs#L12).

In `SentryOptions` people can configure `CaptureFailedRequests`, `FailedRequestStatusCodes` and `FailedRequestTargets` but we don't (yet) provide a mechanism for people to bring their own implementation of `ISentryFailedRequestHandler`. I think if we do that, it should be part of a separate PR... but I'm not sure if we want to or not.